### PR TITLE
Suisin/Update Error Message for new account

### PR DESCRIPTION
**Pull request by suisin-deriv**

### CODE REVIEW

File: src/javascript/app/common/account_opening.js [hunk 1]
```diff
@@ -77,9 +77,11 @@ const AccountOpening = (() => {
     const showResponseError = (response) => {
         getElementById('loading').setVisibility(0);
         getElementById('real_account_wrapper').setVisibility(1);
+        const error_message = response.error.message;
+        const client_error_message = response.error.code.message_to_client;
```
Comment (matin-binary) [inline src/javascript/app/common/account_opening.js:81]: it is better to check if `message_to_client` exists before accessing it.

File: src/javascript/app/common/account_opening.js [hunk 2]
```diff
@@ -293,9 +295,10 @@ const AccountOpening = (() => {
     const handleNewAccount = (response, message_type) => {
         if (response.error) {
             const error_message = response.error.message;
+            const client_error_message = response.error.code.message_to_client;
```
Comment (matin-binary) [inline src/javascript/app/common/account_opening.js:298]: same here

File: src/javascript/app/common/account_opening.js (hunk 1 above)
Comment (suisin-deriv) [reply src/javascript/app/common/account_opening.js:81]: orite 👌

File: src/javascript/app/common/account_opening.js [hunk 3]
```diff
@@ -77,9 +77,11 @@ const AccountOpening = (() => {
     const showResponseError = (response) => {
         getElementById('loading').setVisibility(0);
         getElementById('real_account_wrapper').setVisibility(1);
+        const error_message = response.error.message;
+        const client_error_message = response.error.code.message_to_client;
         const $notice_box = $('#client_message').find('.notice-msg');
         $('#submit-message').empty();
-        $notice_box.text(response.msg_type === 'sanity_check' ? localize('There was some invalid character in an input field.') : response.error.message).end()
+        $notice_box.text(response.msg_type === 'sanity_check' ? localize('There was some invalid character in an input field.') : error_message === null ? client_error_message : error_message).end()
```
Comment (yauheni-kryzhyk-deriv) [inline src/javascript/app/common/account_opening.js:84]: I'd like to suggest you for better readability to move this code  `error_message === null ? client_error_message : error_message` into new variable.
I believe  It is not good practice  to use ternary operator more then once in one place in terms of readability.

and the other thing is: can we use here just these expression: `(error_message || client_error_message)` ?

**Review by matin-binary — APPROVED** [review]

**Review by yauheni-kryzhyk-deriv — APPROVED** [review]

**Pull request merged**